### ISSUE TITLE
After reopening match in the chat messages looks like not read. 

### ISSUE THREAD

**Issue by mrwojciech** [veloce/lichobile#915]
**I'm submitting a 
- bug report

# Bug report

**Current behavior:**
After the game is played with use of chat : private message or auto-message like rematch offered looks like not read when reopening the game. 

**Expected behavior:**

Chat and all messages should stay in status read. 

**Steps to reproduce:**
1. Start game. 
2. Use chat,  write and receive a message. 
3. End the game. 
4. Reopen the game and check chat. 

 

Feature requests should include as much detail as possible:
Please add a time stamp for chat messages so whenever opened I am able to locate the message in time of the game.

**Comment by landry314** [comment]
Yes, I have seen this bug many times. You go back to an old game and chat messages appear to be unread again.
